python/hail/keytable.py

```diff
@@ -1379,5 +1379,113 @@ def range(cls, n, num_partitions=None):
 
         return KeyTable(Env.hc(), Env.hail().keytable.KeyTable.range(Env.hc()._jhc, n, joption(num_partitions)))
 
+    @handle_py4j
+    @record_method
+    @typecheck_method(columns=oneof(strlike, listof(strlike)))
+    def ungroup(self, columns):
```

Comment:
should be varargs, not str or list of str

Reply:
How do I typecheck varargs? Also, this should go on the to-do list for 0.2 as there are other methods that have this interface.

Reply:
fair enough.

You can typecheck varargs as a `tupleof(whatever)`